kernel/yosys.h

```diff
@@ -124,6 +124,8 @@ extern Tcl_Obj *Tcl_ObjSetVar2(Tcl_Interp *interp, Tcl_Obj *part1Ptr, Tcl_Obj *p
 // mingw and msvc include `wingdi.h` which defines a TRANSPARENT macro
 // that conflicts with X(TRANSPARENT) entry in kernel/constids.inc
 #  undef TRANSPARENT
+
```

Comment:
A comment would be useful, perhaps grouping all undefs here under an overall "these Win32 defines conflict with our identifiers" banner.

Reply:
I put a comment before `CONST` and `TRANSPARENT`. Shall I move `NOMINMAX` and `YY_NO_UNISTD_H` down to this group too?

Reply:
Those are different, I think, what you did is sufficient IMO.